exercises/find/problem.md

```diff
@@ -26,6 +26,8 @@ To find a document or documents, one needs to call `find()` on the collection.
 
 Find is a little bit different than what we are used to seeing.
 
+take in mind, retrieve data from `process.argv` is an string, for convert data to integer you should use parseInt().
```

Comment:
Maybe reword to something like:

Keep in mind, `process.argv` is an array of strings. To convert to an integer, you could use `parseInt()`